scripts/zones/Halvung/npcs/qm5.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Area: Halvung
+--  NPC: ??? (Bracelet of verve)
+--  Trade Moblin Oil 
```

Comment:
Last thing. This line has an extra space at the end, making the style check fail.

Reply:
after Moblin Oil?

Reply:
Suggest getting [editorconfig](https://editorconfig.org/) for whatever you use to edit scripts, that will automate things like this so you don;t even have to think about it.